tests/unittests/netmap/multidigraph_to_undirect_test.py

```diff
@@ -57,13 +57,13 @@ def test_layer2_create_directional_metadata_from_nav_graph(self):
 
         # should be the same as
         #  test_edges_length_of_netmap_graph_is_reduced_properly
-        self.assertEqual(
+        assert (
```

Comment:
Since this is a PyUnit test, it really should use the inherited `self.assert*` calls, not a plain assert (unless you would like to rewrite the tests to pure pytest)